README.md

```diff
@@ -1 +1,20 @@
-# Portfolio_Mobile_Version_Skeleton
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Skeleton_Mobile_Version_Portfolio_Website
+
+> I have to create a mobile version of my portfolio website via using Html, Css.
+
+
+## Built With
+
+- Html, CSS
+
+
+
```

Comment:
- [ ] Kindly add `Getting started` section to your `README.md` file, which contains information on how to download and set up your project. This can help non-technical visitors set up and try your project.